etc/conda3_packages-linux-64.yml

```diff
@@ -0,0 +1,97 @@
+name: pinned_20190319
+channels:
+  - defaults
+dependencies:
+  - asn1crypto=0.24.0=py37_0
+  - astropy=3.1.2=py37h7b6447c_0
+  - atomicwrites=1.3.0=py_0
+  - attrs=19.1.0=py_0
+  - blas=1.0=openblas
+  - blosc=1.15.0=hd408876_0
+  - bottleneck=1.2.1=py37h035aef0_1
+  - bzip2=1.0.6=h14c3975_5
+  - ca-certificates=2019.1.23=0
+  - certifi=2019.3.9=py37_0
+  - cffi=1.12.2=py37h2e261b9_1
+  - chardet=3.0.4=py37_1
+  - cryptography=2.6.1=py37h1ba5d50_0
+  - cycler=0.10.0=py37_0
+  - cython=0.29.6=py37he6710b0_0
+  - dbus=1.13.6=h746ee38_0
+  - expat=2.2.6=he6710b0_0
+  - fontconfig=2.13.0=h9420a91_0
+  - freetype=2.9.1=h8a8886c_1
+  - future=0.17.1=py37_0
+  - glib=2.56.2=hd408876_0
+  - gst-plugins-base=1.14.0=hbbd80ab_1
+  - gstreamer=1.14.0=hb453b48_1
+  - h5py=2.9.0=py37h7918eee_0
+  - hdf5=1.10.4=hb1b8bf9_0
+  - icu=58.2=h9c2bf20_1
+  - idna=2.8=py37_0
+  - jpeg=9b=h024ee3a_2
+  - kiwisolver=1.0.1=py37hf484d3e_0
+  - libedit=3.1.20181209=hc058e9b_0
+  - libffi=3.2.1=hd88cf55_4
+  - libgcc-ng=8.2.0=hdf63c60_1
+  - libgfortran-ng=7.3.0=hdf63c60_0
+  - libopenblas=0.3.3=h5a2b251_3
+  - libpng=1.6.36=hbc83047_0
+  - libstdcxx-ng=8.2.0=hdf63c60_1
+  - libuuid=1.0.3=h1bed415_2
+  - libxcb=1.13=h1bed415_1
+  - libxml2=2.9.9=he19cac6_0
+  - lzo=2.10=h49e0be7_2
+  - matplotlib=3.0.3=py37h5429711_0
+  - more-itertools=6.0.0=py37_0
+  - ncurses=6.1=he6710b0_1
+  - nomkl=3.0=0
+  - numexpr=2.6.9=py37h2ffa06c_0
+  - numpy=1.16.2=py37h99e49ec_0
+  - numpy-base=1.16.2=py37h2f8d375_0
+  - openssl=1.1.1b=h7b6447c_1
+  - pandas=0.24.2=py37he6710b0_0
+  - pcre=8.43=he6710b0_0
+  - pip=19.0.3=py37_0
+  - pluggy=0.9.0=py37_0
+  - psutil=5.6.1=py37h7b6447c_0
+  - py=1.8.0=py37_0
+  - pycparser=2.19=py37_0
+  - pyopenssl=19.0.0=py37_0
+  - pyparsing=2.3.1=py37_0
+  - pyqt=5.9.2=py37h05f1152_2
+  - pysocks=1.6.8=py37_0
+  - pytables=3.5.1=py37h71ec239_0
+  - pytest=4.3.1=py37_0
+  - pytest-arraydiff=0.3=py37h39e3cac_0
+  - pytest-astropy=0.5.0=py37_0
+  - pytest-doctestplus=0.3.0=py37_0
+  - pytest-openfiles=0.3.2=py37_0
+  - pytest-remotedata=0.3.1=py37_0
+  - python=3.7.2=h0371630_0
+  - python-dateutil=2.8.0=py37_0
+  - pytz=2018.9=py37_0
+  - pyyaml=5.1=py37h7b6447c_0
+  - qt=5.9.7=h5867ecd_1
+  - readline=7.0=h7b6447c_5
+  - requests=2.21.0=py37_0
+  - scikit-learn=0.20.3=py37h22eb022_0
+  - scipy=1.2.1=py37he2b7bc3_0
+  - setuptools=40.8.0=py37_0
+  - sip=4.19.8=py37hf484d3e_0
+  - six=1.12.0=py37_0
+  - snappy=1.1.7=hbae5bb6_3
+  - sqlalchemy=1.3.1=py37h7b6447c_0
+  - sqlite=3.27.2=h7b6447c_0
+  - tk=8.6.8=hbc83047_0
+  - tornado=6.0.1=py37h7b6447c_0
+  - urllib3=1.24.1=py37_0
+  - wheel=0.33.1=py37_0
+  - wrapt=1.11.1=py37h7b6447c_0
+  - xz=5.2.4=h14c3975_4
+  - yaml=0.1.7=had09818_2
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - deprecated==1.2.5
+prefix: /home/gcomoret/anaconda3/envs/pinned_20190319
```

Comment:
Does this prefix entry do anything?

Reply:
No, the name and the prefix are not considered when creating an environment from the yml file